exporter.go

```diff
@@ -901,12 +901,14 @@ func (e *Exporter) extractLatencyMetrics(ch chan<- prometheus.Metric, c redis.Co
 	if reply, err := doRedisCmd(c, "LATENCY", "LATEST"); err == nil {
```

Comment:
make this `redis.Values(doRedisCmd( ... ` and then you won't need line 903 at all and can do the range over `reply` right away. 

While you're at it, you could improve two more things:
- use another call to `redis.Values()` (with err check) in line 905 instead of the current cast
- move eventName into the loop to reduce the scope

Reply:
I made the changes you suggested.  Is that what you were thinking?

Reply:
Yeah, this looks good, thanks for changing it.